packages/block-library/src/list/deprecated.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * WordPress dependencies
+ */
+import { RichText, useBlockProps } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import migrateFontFamily from '../utils/migrate-font-family';
+
+const v1 = {
+	attributes: {
+		ordered: {
+			type: 'boolean',
+			default: false,
+			__experimentalRole: 'content',
+		},
+		values: {
+			type: 'string',
+			source: 'html',
+			selector: 'ol,ul',
+			multiline: 'li',
+			__unstableMultilineWrapperTags: [ 'ol', 'ul' ],
+			default: '',
+			__experimentalRole: 'content',
+		},
+		type: {
+			type: 'string',
+		},
+		start: {
+			type: 'number',
+		},
+		reversed: {
+			type: 'boolean',
+		},
+		placeholder: {
+			type: 'string',
+		},
+	},
+	supports: {
+		anchor: true,
+		className: false,
+		typography: {
+			fontSize: true,
+			__experimentalFontFamily: true,
+		},
+		color: {
+			gradients: true,
+			link: true,
+		},
+		__unstablePasteTextInline: true,
+		__experimentalSelector: 'ol,ul',
+	},
```

Comment:
I'm now thinking a bit more about structuring deprecations better and less redundant 🤔 

How about we do

```suggestion
import blockConfig from './block.json';

const {
	attributes: currentAttributes,
	supports: currentSupports,
} = blockConfig;

const v1 = {
	attributes: currentAttributes,
	supports: currentSupports,
```

This pattern should work for all blocks deprecations that update to the block's current attributes format (and block supports), I think.

Reply:
Trying out in b60ba0e998. We can revert if needed.